ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -4644,13 +4644,7 @@
     "CaseTypeID": "CIVIL",
     "CaseFieldID": "CaseAccessCategory",
     "AccessControl": [
-      {
-        "UserRoles": [
-          "caseworker-civil-solicitor"
-        ],
-        "CRUD": "CRUD"
-      },
-      {
+     {
```

Comment:
We need solicitor auths for this field, as the callback is a user event. This is where the CaC gets set, as a solicitor user:

https://github.com/hmcts/civil-service/blob/master/src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandler.java#L424

Reply:
Harry, I removed that because this field need to be only seen to CaseWorker, After removing access control I created one demages and one spec claim okay locally, so I thought removing solicitor access should be okay as I could create claim locally successfully.

Reply:
How / Where is the Case Access Category getting set?